lib/facility.rb

```diff
@@ -1,14 +1,84 @@
+require './lib/vehicle'
+require './lib/registrant'
+require 'date'
+
 class Facility
-  attr_reader :name, :address, :phone, :services
+  attr_reader :name, :address, :phone, :collected_fees, :registered_vehicles
+  attr_accessor :services
 
-  def initialize(name, address, phone)
-    @name = name
-    @address = address
-    @phone = phone
+  def initialize(hash, collected_fees = 0)
+    @name = hash[:name]
+    @address = hash[:address]
+    @phone = hash[:phone]
     @services = []
+    @collected_fees = collected_fees
+    @registered_vehicles = []
   end
 
-  def add_services(service)
+  def add_service(service)
     @services << service
+    @services = @services.uniq
   end
```

Comment:
This looks great! Alternatively, we could also only shovel the service into the `@service` array IF it didn't already exist in the array. 

Something like: 

```ruby
@services << service if !@services.include?(service)
```

Reply:
To be clear, either way would be totally fine.. but sometimes it's fun to see a different approach ;)